test/fixtures/flow.expected.txt

```diff
@@ -0,0 +1,21 @@
+Error ---------------------------------------------------------------------------- test/fixtures/flow.input.js.flow:26:9
+
+Cannot call `Pbf` with `'wrong type'` bound to `buf` because:
+ - Either string [1] is incompatible with `ArrayBuffer` [2].
+ - Or string [1] is incompatible with `Uint8Array` [3].
+
+   test/fixtures/flow.input.js.flow:26:9
+   26| new Pbf('wrong type');
+               ^^^^^^^^^^^^ [1]
+
+References:
+   index.js.flow:6:23
+    6|     constructor(buf?: ArrayBuffer | Uint8Array): Pbf;
+                             ^^^^^^^^^^^ [2]
+   index.js.flow:6:37
+    6|     constructor(buf?: ArrayBuffer | Uint8Array): Pbf;
+                                           ^^^^^^^^^^ [3]
+
+
+
+Found 1 error
```

Comment:
Thinking about this further, I'm not comfortable with this approach to testing the existence of flow types.

- It will constantly break between flow versions when the output changes in subtle ways.
- It doesn't really test whether the library provides all the necessary flow types — only a few selected ones in `flow.input.js.flow`.

Instead, is there a way to test flow through the actual tests that exercise all the code paths in the library?